code/index.html

```diff
@@ -1,30 +1,227 @@
+            <fieldset class="container-form">
+                <legend>
+                    <h2>Harvest Dream</h2>
+                </legend>
+                <div class="form-group">
+                    <label for="firstName">First name*</label>
+                    <input 
+                    id="firstName" 
+                    type="text" 
+                    placeholder="First name" 
+                    name="firstNameValue" 
+                    required
+                    >
+                </div>
+                <div class="form-group">
+                    <label for="lastName">Last name*</label>
+                    <input 
+                    id="lastName" 
+                    type="text" 
+                    placeholder="Last name" 
+                    name="lastNameValue" r
+                    equired
+                    >
+                </div>
+                <div class="form-group">
+                    <label for="password">Password</label>
+                    <input 
+                    id="password" 
+                    type="password" 
+                    name="passwordValue"
+                    >
+                </div>
+
+    <!--CHECKBOX-->
+
+                <p>Choose themes</p>
+                <div wrapper-row>
+                
+                    <label class="checkbox-container">This
+                        <input 
+                        id="this"
+                        type="checkbox" 
+                        name="this"
+                        checked
+                        >
+                        <span class="custom-checkbox"></span>
+                    </label>
+                
+                    <label class="checkbox-container">That
+                        <input 
+                        id="that"
+                        type="checkbox" 
+                        name="that"
+                        >
+                        <span class="custom-checkbox"></span>
+                    </label>
+
+                    <label class="checkbox-container">Those
+                        <input 
+                        id="those"
+                        type="checkbox" 
+                        name="those"
+                        >
+                        <span class="custom-checkbox"></span>
+                    </label>
+                </div>
+                
+                
+    <!--RADIOBUTTONS-->            
+                               
+                <div class="wrapper-col">
+                    <p>Choose your size</p>
+                    
+                    <label class="radiobutton-container">Small
+                    <input 
+                    type="radio" 
+                    id="small" 
+                    name="answer" 
+                    value="small"
+                    >
+                    <span class="custom-radio"></span>                    
+                    </label>
```

Comment:
To structure this and all of the similar tags, try to indent the input one level since it's wrapped with a label 👋